### PR TITLE
Added your organizations on index view

### PR DESCRIPTION
**Pull request by Cheshiriks**
- Added your organizations on index view
- Added controller for count if vulnerabilities

Снимок экрана 2023-08-08 в 00 00 45

### CODE REVIEW

**Comment by sanyavertolet** [comment]
To my mind, `indexView` is now a bit narrow. Text columns are so small while there are quite many words. Moreover, your organisation card looks too small and not that expressive. Finally, paddings should be fixed as, for example, padding between `ProfileSettings` button and your title is less then between the title and organisation list

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt [hunk 1]
```diff
@@ -29,12 +31,10 @@ import kotlinx.datetime.toLocalDateTime
 
 // FixMe: List of organizations where user included, if not - link to creation of organization
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt:32]: Is this one also done?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt [hunk 2]
```diff
@@ -158,5 +168,38 @@ val cardUser: FC<IndexViewProps> = FC { props ->
                     }
             }
         }
+
+        div {
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt:172]: I'd also wrap the title and organisations with a `div` and set paddings in high-level `div`

**Review by sanyavertolet — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt]: Removed

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt (hunk 2 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt:171]: Fixed

**Review by akuleshov7 — APPROVED** [review]

**Pull request merged**